migrations/m251020_093403_create_table_pokemon_instances.php

```diff
@@ -7,7 +7,8 @@
  *
  * stores a pokemon instance with data about that species, and how it is specifically configured by the user
  *
- * @todo add validation for team_id XOR box_id
+ * -
+ * @todo add validation for custom constraint 'team_id XOR box_id'
```

Comment:
If this hasn't been resolved yet, we can create an issue in GitHub to resolve it later?

Otherwise this TODO might live here forever.